Sources/SpeziAccount/AccountNotifyConstraint.swift

```diff
@@ -32,4 +33,15 @@ public protocol AccountNotifyConstraint: Standard {
     ///
     /// For more information refer to ``AccountNotifications/Event``.
     func respondToEvent(_ event: AccountNotifications.Event) async
+    
+    /// Notifies the standard that the currently logged-in account is about to be logged out.
+    ///
+    /// - Note: This function will be folded into the ``AccountNotifications/Event`` enum in a future release.
+    func willLogOut(_ details: AccountDetails) async
```

Comment:
Would be great if can also summarize all these future-looking elements in one GitHub issue to document things there 👍

Reply:
Sounds great; thank you!